src/bitcoin/P2tr.mo

```diff
@@ -0,0 +1,33 @@
+import Types "./Types";
+import Ecdsa "../ecdsa/Ecdsa";
+import Script "./Script";
+import Segwit "../Segwit";
+import Result "mo:base/Result";
+
+module {
+    type PublicKey = Ecdsa.PublicKey;
```

Comment:
Aren't Taproot public keys always Schnorr not ECDSA?

Reply:
yes, that was dead code, removed